Scripts/StixParser/StixParser.py

```diff
@@ -178,29 +195,48 @@ def get_indicators(indicators):
         indicators (dict or list): STIX-formatted entry
 
     Returns:
-        dict: in the format of:
+        (dict, dict): in the format of:
+        (
             {
                 "File": [],
                 "IP": [],
                 "Domain": [],
                 "URL": []
+            },
+            {
+                "<key>": "<stix entry>"
             }
+            )
     """
 
     def indicators_parser(stix_indicator):
```

Comment:
can we take that function out?

Reply:
problem with taking it out. anyway, it'll be used only in this function. you can use this function with sending input to the parent function (which only checks if we got dict or list